robotframework_reportportal/result_visitor.py

```diff
@@ -27,7 +27,10 @@ def start_suite(self, suite):
             'source': suite.source,
             'suites': suite.suites,
             'tests': suite.tests,
-            'totaltests': suite.statistics.all.total,
+            try:
+                'totaltests': suite.statistics.all.total,
+            except:
```

Comment:
Too broad exception. You can either declare what exception you want to catch or use hasattr method:
```
'totaltests' : suite.statistics.all.total if hasattr(suite.statistics.all.total, 'total') else suite.statistics.total
```

Reply:
Updated script to use `hasaatr` instead of exception. Not really aware on flag case through code!

Reply:
I apologize but I made mistake in my original example - I've updated my comment. Were checking if all exist, not total

Reply:
Thank you! update syntax. Please review